projects/WeTek_Hub/linux/linux.aarch64.conf

```diff
@@ -34,7 +39,7 @@ CONFIG_CROSS_COMPILE=""
 CONFIG_LOCALVERSION=""
 # CONFIG_LOCALVERSION_AUTO is not set
 CONFIG_DEFAULT_HOSTNAME="@DISTRONAME@"
-CONFIG_SWAP=y
+# CONFIG_SWAP is not set
```

Comment:
It is only disguised as enabled in the current `linux.aarch64.conf `-file, at compile time it will be disabled due to the `options`-file. So I guess we should make a PR to enable swap?